Maquetado/scss/styles.scss

```diff
@@ -60,28 +61,39 @@ body {
 
 .techno {
   align-items: center;
-  border: 1px solid $green;
-  border-radius: 5px;
   display: flex;
   flex-direction: column;
   justify-content: center;
-  margin: 100px 20px 10px;
   padding: 20px;
-  position: relative;
-  max-width: 200px;
+  max-width: 222px;
   width: 100%;
+}
 
-  &::before {
-    background-color: $green;
-    border-radius: 5px;
-    content: '';
-    height: 100%;
-    left: 10px;
-    position: absolute;
-    top: 10px;
-    width: 200px;
-    z-index: -1;
-  }
+.react-ruby {
+  border-radius: 5px;
```

Comment:
I know it has a radius on the image but it was a typo :stuck_out_tongue_closed_eyes: no need for this